### PR TITLE
[docker-base]: Enabled common debug packages installation.

### PR DESCRIPTION
**Pull request by antonpatenko**
The next debug packages are to be installed through docker-base:
 - gdb
 - vim
 - openssh-client
 - sshpass

### CODE REVIEW

File: dockers/docker-base/Dockerfile.j2 [hunk 1]
```diff
@@ -33,9 +33,16 @@ RUN apt-get -y purge                    \
...
+## Install common debug-packages
+RUN apt-get -y install \
+{% for dbg_pkg in docker_base_dbgs.split(' ') -%}
+    {{ dbg_pkg }}{{' '}}
+{%- endfor %}
+{% endif %}
 
+## Clean up
+RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
+## Note: NO removing /var/lib/apt/lists/*, shared by all derived images
```
Comment (lguohan) [inline dockers/docker-base/Dockerfile.j2]: we should remove, the derived images will do apt-get update again. We can save the baseimage iamge if we remove.

File: rules/config [hunk 2]
```diff
@@ -31,3 +31,6 @@ USERNAME = admin
 
 # PASSWORD - password for installer build
 PASSWORD = YourPaSsWoRd
+
+# SONIC_CONFIG_DEBUG - install debug packages
+SONIC_CONFIG_DEBUG = n
```
Comment (lguohan) [inline rules/config]: can you follow other comments?

 # Uncomment next line to enable:
 # SONIC_CONFIG_DEBUG = y

File: dockers/docker-base/Dockerfile.j2 [hunk 3]
```diff
@@ -36,9 +36,12 @@ RUN apt-get -y purge                    \
     exim4-config                        \
     exim4-daemon-light
 
-## Clean up apt
-## Remove /var/lib/apt/lists/*, could be obsoleted for derived images
-RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y;      \
-    rm -rf /var/lib/apt/lists/*;                                        \
-    rm -rf /tmp/*;
```
Comment (lguohan) [inline dockers/docker-base/Dockerfile.j2]: we should keep all these.

**Comment by qiluo-msft** [comment]
:shipit:

**Pull request merged**